tests/testChain.cpp

```diff
@@ -183,7 +185,7 @@ TEST(Chain, ChainConstraint) {
 
   // Create VectorExpressionEquality Constraint
   auto constraint = VectorExpressionEquality<3>(expression, tolerance);
-  Vector3 expected_values(1, 1.9, 1.3);
+  Vector3 expected_values(1, 0.46, 0.82);
```

Comment:
Say where these hardcoded values come from. If from output (and sanity checked by you, obviously), add comment `// regression`.

Reply:
done